pwiz_tools/Skyline/Model/AbstractMassListExporter.cs

```diff
@@ -224,9 +224,14 @@ private void NextFile(FileIterator fileIterator)
             fileIterator.NextFile();
         }
 
-        public string GetCompound(PeptideDocNode nodePep, TransitionGroupDocNode nodeGroup)
+        public string GetCompound(PeptideDocNode nodePep, TransitionGroupDocNode nodeGroup, bool showLabelType = false)
         {
-            return nodePep.Peptide.IsCustomMolecule ? nodeGroup.CustomMolecule.InvariantName : Document.Settings.GetModifiedSequence(nodePep).Sequence;
+            string compoundName = nodePep.Peptide.IsCustomMolecule
+                ? nodeGroup.CustomMolecule.InvariantName
+                : Document.Settings.GetModifiedSequence(nodePep).Sequence;
+            if (showLabelType)
+                compoundName = string.Format(@"{0} ({1})", compoundName, nodeGroup.TransitionGroup.LabelType);
+            return compoundName.ToDsvField(FieldSeparator, FieldSeparatorReplacement);

```

Comment:
It looks like all the other code in the project that was using "GetCompound" was already escaping it, doing things like this in "ThermoMassListExporter.WriteTransition":
`writer.WriteDsvField(GetCompound(nodePep, nodeTranGroup), FieldSeparator, FieldSeparatorReplacement);`

or even this code in ThermoQuantivaMassListExporter.WriteTransition which does not specify a replacement character and would theoretically be relying on quotes being correctly interpreted:
`writer.WriteDsvField(compound, FieldSeparator);`